src/dbt_client/dbtCoreIntegration.ts

```diff
@@ -909,6 +909,33 @@ export class DBTCoreProjectIntegration
     }
   }
 
+  findPackageVersion(packageName: string) {
+    if (!this.packagesInstallPath || !packageName) {
+      return;
+    }
+
+    const dbtProjectYmlFilePath = path.join(
+      this.packagesInstallPath,
+      packageName,
+      "dbt_project.yml",
+    );
+    if (!existsSync(dbtProjectYmlFilePath)) {
+      return;
```

Comment:
Is this not an error? this means package is not installed.

Reply:
This is not an error. If the file does not exist, then user does not have this package installed and we dont have to use it.